pg_auth_mon--1.0.sql

```diff
@@ -14,11 +14,16 @@ CREATE FUNCTION pg_auth_mon(
     OUT last_successful_TS timestampTz,
     OUT total_hba_conflicts   int,
     OUT other_auth_failures    int,
-    OUT last_failed_TS  timestampTz
+    OUT last_failed_TS  timestampTz,
+    OUT initial_rolename name
```

Comment:
It is not enough just to edit the existing file, one has to provide the upgrade path from the old version to the new one.